packages/react-composites/src/composites/ChatComposite/styles/Chat.styles.ts

```diff
@@ -53,7 +53,8 @@ export const chatArea = mergeStyles({
 export const chatWrapper = mergeStyles({
   height: '100%',
   width: '100%',
-  overflow: 'hidden'
+  overflow: 'hidden',
+  zIndex: '5'
```

Comment:
We need to adjust z-index, otherwise messages lay on top of participant pan
<img width="200" alt="Screen Shot 2022-05-31 at 12 35 22 PM" src="https://user-images.githubusercontent.com/97124699/171270878-fb388f4b-4ae7-4d11-864a-5437d7ad0c96.png">
e

Reply:
Curious how we arrived at the value "5" for zIndex. How would it behave if the zIndex here was 1, 2, 3 or 4?

Reply:
done